### DISCUSSION TITLE
Is there a way to install this fork with Home Assistant Core

### DISCUSSION THREAD

**Discussion by bakercj** [sharkiqlibs/sharkiq#8] [General]
I am unclear on if this is an integration is available for Home Assistant Core/OS. Can someone confirm whether it is supported and if so, where should the pip install be run?

I appreciate the help.

**Comment by JeffResc** [comment]
I don't personally use Home Assistant Core, so I might not be the right person to answer this question, but this fork is included in Home Assistant Core's latest version, 2022.4 [as shown in the changelog](https://www.home-assistant.io/changelogs/core-2022.4), so you should be able to install it like you would any other update to the core.

Does that help answer your question?

**Comment by bakercj** [comment]
Hm. I looked through the changelog specifically for this change and missed it. Thanks. I guess I am still experiencing the same issue as pre 2022.4 with a Unexpected error when attempting to authenticate. Any ideas?

**Comment by bakercj** [comment]
Scratch that. I dont know what changed but suddenly it worked for me. Thank you for your assistance.

**Comment by funkybunch** [comment]
Yeah it might require a reload after upgrading to 2022.4.

As far as core vs OS, Home Assistant core is installed on top of Home Assistant OS.  For all intensive purposes, they're one in the same for updates to this integration.
